### PR TITLE
Add PHPBU role

### PR DESCRIPTION
**Pull request by tborealis**

### CODE REVIEW

File: phpbu/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+- block:
```
Comment (lopsided) [inline phpbu/tasks/main.yml]: Not sure I like this style very much, but only aesthetically. No need to change if you like it

**Review by lopsided — COMMENTED** [review]
Didn't mean to start a review :-P

File: phpbu/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    get_url:
+      url: '{{ phpbu_config_schema_url }}'
+      dest: '{{ phpbu_lib_dir }}/phpbu.xsd'
+      owner: root
+      group: phpbu
+      mode: '0664'
+
+  - name: Configure
+    template:
+      src: templates/phpbu/phpbu.xml
```
[the remark this answers is not in the document]
Comment (TomAdam) [reply phpbu/tasks/main.yml]: The config for PHPBU isn't particularly reusable between installs. I take your point, but I think in this case the docs are better than anything that could be supplied as an example.

File: phpbu/tasks/main.yml (hunk 1 above)
Comment (TomAdam) [reply phpbu/tasks/main.yml]: I'm not a massive fan either. Do you think it would be better split into anther file and an include?

**Pull request closed without merging**